### PR TITLE
Add sidebar docs to extensibility docs

### PR DESCRIPTION
**Pull request by IreneStr**
Add sidebar docs to the extensibility docs.

### CODE REVIEW

File: docs/extensibility.md [hunk 1]
```diff
@@ -120,6 +120,35 @@ wp.hooks.addFilter(
...
+### Activating a sidebar
+
+`wp.editPost.activateSidebar( name: string )`
+
+This method takes the `name` of the sidebar you'd like to open. That sidebar should have been registered beforehand using the `registerSidebar` method.
+
+**Example**
+
+```js
+wp.data.activateSidebar( 'my-plugin/my-custom-sidebar' );
```
Comment (youknowriad) [inline docs/extensibility.md]: Just tried this and it looks like there's a bug, this doesn't work unless the default sidebar of the editor is opened.

Comment (youknowriad) [reply docs/extensibility.md]: And there's a typo here :) `wp.data`

**Comment by IreneStr** [comment]
@youknowriad Thanks for pointing out the typo. I've fixed it (also in the docs where I copied it from).

The other issue you're pointing out is a known issue we discussed in the sidebar PR comments but moved to a next iteration. Do you want me to describe in this docs that the sidebar is not working yet in that case?

**Comment by youknowriad** [comment]
>  Do you want me to describe in this docs that the sidebar is not working yet in that case?

No, it's fine if it's a known issue that will be solved at some point.

**Review by youknowriad — APPROVED** [review]
LGTM 👍

**Pull request merged**